### ISSUE TITLE
train and test split

### ISSUE THREAD

**Issue by sophie-isobel** [Tencent/MedicalNet#15]

**Comment by sophie-isobel** [comment]
I would like to use your pretrained ResNet for transfer learning on the BraTS dataset, but since 90% of it was used in training the pretrained model, I would need to know which scans were used in order to exclude them from my test set. Is this information available? thanks.

**Comment by DaiyangLiu** [comment]
the same question~~
